burn-import/src/burn/node/base.rs

```diff
@@ -146,6 +149,7 @@ impl<PS: PrecisionSettings> Node<PS> {
             Node::MaxPool2d(_) => "max_pool2d",
             Node::Reshape(_) => "reshape",
             Node::Unary(unary) => unary.kind.as_str(),
+            Node::Unsqueeze(_) => "unsqueeze_dims",
```

Comment:
We should just name `unsqueeze` instead. These are string representation of enums.